Make.inc

```diff
@@ -693,6 +693,15 @@ else
 ISX86:=0
 endif
 
+# clang needs this flag to compile CRC instructions (used conditionally with detection)
+ifneq (,$(findstring aarch64,$(ARCH)))
+ifeq ($(USECLANG),1)
+ifeq ($(MARCH),)
+JCFLAGS += -mcrc
```

Comment:
Does clang not support the `target` attribute? This is still wrong if it doesn't though since IIUC it'll allow all the code to use the crc extention.

Reply:
It does, but it doesn't recognize `+crc` in the target.

Yes, it'll allow all code to use it, but I don't think LLVM would ever use CRC automatically. Is there a good way to set flags per file in your makefile?

Reply:
Does it recognize anything else? e.g. `armv8-a+crc`.

Reply:
I mean, I was talking about `armv8-a+crc`, not bare `+crc` of course :D

https://bugs.chromium.org/p/chromium/issues/detail?id=554632

Apparently that was "fixed", but I don't know where. Not in regular 6.0.1.

Reply:
Yeah, exactly. Are there any non-crc32 cores other than X-Gene 1?

Reply:
I do have an X-Gene 1 so that's one. I couldn't find a feature set for a35 and the arm page didn't mention crc and I didn't find the manual last time I checked.

Reply:
Also possibly denver 1, I found no info about it...

Reply:
Hm, actually — clang is fine with the per function `+crc`… except there was an extra plus sign in `__attribute__((target("+crc")))` — clang adds its own plus and you get `++`:

```
'++crc' is not a recognized feature for this target (ignoring feature)
```

`__attribute__((target("crc")))` seems to work